src/packages/@ncigdc/utils/constants.js

```diff
@@ -222,3 +222,21 @@ export const IMPACT_SHORT_FORMS = {
     unknown: 'UN',
   },
 };
+
+export const FAKE_USER = process.env.ALLOW_FAKE_USER
```

Comment:
can you prefix `ALLOW_FAKE_USER` with `REACT_APP_` to be consistent with the other process.env vars